### PR TITLE
C_CPP :- Using Of stack for implementation of queue in document form

### PR DESCRIPTION
**Pull request by Pranshu321**
<hr>

## Description 📜

In this, I have explained how the stack can be used to implement a queue in 3 different ways, and with their time and complexities, all methods are explained in great detail.
Issue number closed :- #8072

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: C_CPP/Data Structures/Stack/Queue_Using_Stack.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
+# Implementation Queue Using Stack
```
Comment (deeqakkk) [inline C_CPP/Data Structures/Stack/Queue_Using_Stack.md:1]: fix grammar mistake and add "of"

File: C_CPP/Data Structures/Stack/Queue_Using_Stack.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+A queue is a data structure which follows FIFO operation (which means First In First Out) and if we talk about Stack it is the data structure which follows LIFO (which means Last In First Out) and we can implement queue using the stack , There are three by which we can implement this in each method any one operation enqueue or dequeue become costly , Methods are :
```
Comment (deeqakkk) [inline C_CPP/Data Structures/Stack/Queue_Using_Stack.md:2]: `there are three...` fix this as well, what three you want to convey

**Review by deeqakkk — CHANGES_REQUESTED** [review]
Pranshu, your content is good, just fix some typos and tag me

**Comment by Pranshu321** [comment]
All the changes have been done please review it again @deeqakkk.

**Review by deeqakkk — APPROVED** [review]
Content is good Pranshu👏💯.
@Suvraneel its good to be merged.

**Comment by Suvraneel** [comment]
@Pranshu321  For one this is a duplicate.... And in the wrong directory
But Idk how the last one went past merge... But its kinda broken....
@Pranshu321  this is an exception but that one needs replacing.... (here](https://github.com/girlscript/winter-of-contributing/blob/C_CPP/C_CPP/Data%20Structures/Queue/implementation_of_queue_using_stack.md)

**Comment by Pranshu321** [comment]
> @Pranshu321 For one this is a duplicate.... And in the wrong directory
> 
> We already have contribution on this topic.... This one's great though... But Idk how the last one went past merge... But its kinda broken.... So as an exception, @Pranshu321 That one needs replacing.... [here](https://github.com/girlscript/winter-of-contributing/blob/C_CPP/C_CPP/Data%20Structures/Queue/implementation_of_queue_using_stack.md)
> 
> Replace that old file & change readme links accordingly... And yes great documentation 🚀 Good work

Thanx @Suvraneel for appreciating my work, as the changes that you said will make this a little messy so I have generated new in which all the changes that are required from deleting the file to changes links all the thing will be done in that PR, please merge that PR in one go after cross-checking it as it's in the right folder or not.

**Comment by Suvraneel** [comment]
@Pranshu321  Done... Closing this one ... The latest one is merged

**Pull request closed without merging**